pkg/collector/process.go

```diff
@@ -59,6 +60,7 @@ type CollectorInput struct {
 	MaxBufferSize uint16
 	TemplateTTL   uint32
 	IsEncrypted   bool
+	CACert        []byte
 	ServerCert    []byte
 	ServerKey     []byte
```

Comment:
should we group them in a separate structure for this and use it for both exporter and collector by renaming ServerCert/ClientCert -> Cert and other input to Key? 
Not sure if it changes any meaning for the respective processes.

Ok to move on with a TODO as this is a minor comment for cleaner code.

Reply:
I added a TODO comment here for changes later.